.github/workflows/pulumi-aws.yml

```
@@ -87,30 +102,40 @@ env:
+    timeout-minutes: 90
     env:
       IAC_GITHUB_API_TOKENS: ${{ secrets.iac-github-api-tokens }} # only used in non-enterprise situations when not using full fledged Secrets Manager
     steps:
       - name: Checkout code
-        uses: actions/checkout@v4.2.2
+        uses: actions/checkout@v6.0.2
         with:
           ref: ${{ inputs.CHECKOUT_REF }}
 
       - name: Install latest versions of python packages
         uses: ./.github/actions/install_deps
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/install_deps": line 4: unexpected key "type" for definition of input "skip-installing-pulumi-cli"

(action)